### PR TITLE
OpenJDK 11

### PR DESCRIPTION
**Pull request by VonUniGE**
- [x] Have you followed the [guidelines for contributing](https://github.com/Linuxbrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Linuxbrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?
- [ ] Have you included the output of `brew gist-logs <formula>` of the build failure if your PR fixes a build failure. Please quote the exact error message.

### CODE REVIEW

File: Formula/openjdk@11.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+class OpenjdkAT11 < Formula
+  desc "Java Development Kit"
+  homepage "http://jdk.java.net/11/"
```
Comment (retokromer) [inline Formula/openjdk@11.rb]: use `https` rather than `http`

File: Formula/openjdk@11.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  # tag "linuxbrew"
+
+  version "11+28"
+  if OS.mac?
+    url "http://java.com/"
```
Comment (retokromer) [inline Formula/openjdk@11.rb]: use `https` rather than `http`

**Comment by VonUniGE** [comment]
No one interrested in OpenJDK 11?

**Comment by iMichka** [comment]
Sorry for the delay. Thanks for reminding me :)

I think there was a comment about `http`/`https` still needs to be addressed.
Can you also rebase this?

**Comment by VonUniGE** [comment]
Ok, missing https added, branch squashed and rebased.

**Comment by iMichka** [comment]
Thanks @VonUniGE!

**Pull request closed without merging**